src/crs_linter/cli.py

```diff
@@ -313,6 +329,34 @@ def main():
     parsed = read_files(files)
     txvars = {}
 
+    if args.tests is not None:
+        # read existing tests
+        testlist = glob.glob(args.tests)
```

Comment:
Ensure that `args.tests` is really a path, then use the proper glob pattern.
This is why it doesn't work on GH BTW, because `tests/regression/tests/*` resolves only to the *directories* containing the tests.
```suggestion
        testlist = glob.glob(f"{args.tests}/**/*.y[a]ml")
```